packages/account/src/Components/poi/poi-confirm-with-example-form-container/poi-confirm-with-example-form-container.tsx

```diff
@@ -0,0 +1,198 @@
+import React from 'react';
+import classNames from 'classnames';
+import { Form, Formik, FormikHelpers } from 'formik';
+import { GetSettings } from '@deriv/api-types';
+import { Checkbox, HintBox, Loading, Text } from '@deriv/components';
+import { Localize, localize } from '@deriv/translations';
+import { filterObjProperties, isMobile, toMoment, validLength, validName, WS } from '@deriv/shared';
+import FormBody from 'Components/form-body';
+import LoadErrorMessage from 'Components/load-error-message';
+import PersonalDetailsForm from 'Components/forms/personal-details-form';
+import { validate, makeSettingsRequest } from 'Helpers/utils';
+
+type TValues = { [p: string]: string };
+
+type TRestState = {
+    api_error: string;
+    show_form: boolean;
+    errors?: boolean;
+    form_initial_values: TValues;
+    changeable_fields: string[];
+};
+
+type TPoiConfirmWithExampleFormContainer = {
+    account_settings: GetSettings;
+    getChangeableFields: () => string[];
+    onFormConfirm?: () => void;
+};
+
+const PoiConfirmWithExampleFormContainer = ({
+    account_settings,
+    getChangeableFields,
+    onFormConfirm,
+}: TPoiConfirmWithExampleFormContainer) => {
+    const [is_loading, setIsLoading] = React.useState(true);
+    const [checked, setChecked] = React.useState(false);
+    const [rest_state, setRestState] = React.useState<TRestState>({
+        show_form: true,
+        form_initial_values: { first_name: '', last_name: '', date_of_birth: '' },
+        changeable_fields: [],
+        api_error: '',
+    });
+
+    React.useEffect(() => {
+        initializeFormValues();
+        // eslint-disable-next-line react-hooks/exhaustive-deps
+    }, [account_settings]);
+
+    const onSubmit = async (values: TValues, { setStatus, setSubmitting }: FormikHelpers<TValues>) => {
+        if (checked) return;
+        setStatus({ error_msg: '' });
+        const request = makeSettingsRequest(
+            values,
+            rest_state?.changeable_fields ? [...rest_state.changeable_fields] : []
+        );
+        const data = await WS.setSettings(request);
```

Comment:
@yauheni-deriv And why not `useRequest`? :D

Reply:
thanks @farzin-deriv for the advice. As discussed, now we have to implement to many changes in account package and few in cfd. 
we will try to implement it as a technical debt during upcoming sprints